packaging/flavorgen/flavors/generators.go

```diff
@@ -305,6 +306,17 @@ func defaultControlPlaneComponent() bootstrapv1.ControlPlaneComponent {
 	}
 }
 
+func defaultPciDevices() []infrav1.PCIDeviceSpec {
+	deviceID := int32(env.VSpherePCIDeviceID)
```

Comment:
I don't think we should add PCI devices as a mandatory thing in our template, we could a new flavor in flavorgen and only add it there.